Apromore-Custom-Plugins/Account-Portal-Plugin/src/main/resources/account.properties

```diff
@@ -0,0 +1,6 @@
+passwordTooShort = New password must be at least 6 characters long.
```

Comment:
Pass the value 6 as a parameter, using java.text.MessageFormat to localize it.

Reply:
Will create a separate ticket to pull this from application.properties